container/apptabs.go

```diff
@@ -106,9 +106,16 @@ func (c *AppTabs) CurrentTabIndex() int {
 	return c.current
 }
 
+// ExtendBaseWidget is used by an extending widget to make use of BaseWidget functionality.
+//
+// Deprecated: Support for extending containers is being removed
+func (c *AppTabs) ExtendBaseWidget(wid fyne.Widget) {
```

Comment:
I thought we said, on the call yesterday, that we wouldn't add "new" deprecated methods?

Reply:
Yes we did. I have not added the new no-op `ExtendBaseWidget` in `Scroll` where it would be required.
These methods are not "new" they are wrappers so that I can mark Deprecated.
Obviously we don't want to set it Deprecated on `widget.BaseWidget` which is what it overrides now.

Reply:
Ah, I see. Thanks for the clarification.